order-api/src/main/java/com/example/orderapi/interfaces/presentation/OrderController.java

```diff
@@ -0,0 +1,21 @@
+package com.example.orderapi.interfaces.presentation;
+
+import com.example.orderapi.application.service.OrderService;
+import com.example.orderapi.interfaces.presentation.request.OrderRequest;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+@RequestMapping("/api/v1")
+@RequiredArgsConstructor
+@RestController
+public class OrderController {
+
+    private final OrderService orderService;
+
+    @ResponseStatus(HttpStatus.CREATED)
+    @PostMapping("/orders")
+    public void crateOrder(@RequestBody OrderRequest reqest) {
```

Comment:
```suggestion
    public void crateOrder(@Valid @RequestBody OrderRequest reqest) {
```

별건 아니지만 body에 들어오는 값을 검증하는게 추가되면 좋을것 같아요~